llvm/test/CodeGen/AArch64/streaming-func-no-sme.ll

```diff
@@ -0,0 +1,7 @@
+; RUN: not llc -mtriple aarch64-none-linux-gnu %s 2>&1 | FileCheck %s
+
+; CHECK: LLVM ERROR: streaming SVE functions require SME
+define void @streaming(i64 noundef %n) "aarch64_pstate_sm_enabled" nounwind {
+entry:
+  ret void
+}
```

Comment:
```suggestion
}

```

Reply:
I think this works for now (this case should be checked in the frontend too).